lib/Bio/EnsEMBL/DataCheck/Checks/MultipleSeqRegions.pm

```diff
@@ -35,6 +35,28 @@ use constant {
   TABLES      => ['structural_variation_feature', 'variation_feature']
 };
 
+sub skip_tests {
+  my ($self) = @_;
+
+  my $desc_dna_dba = 'Core database found';
+  my $dna_dba = $self->get_dna_dba();
+  my $pass = ok(defined $dna_dba, $desc_dna_dba);
+
+  # Note that if $pass is false, we will still execute the 'tests'
+  # method; which is either not required, or will need to done again,
```

Comment:
small typo: will need to **be** done again

Reply:
Thanks, fixed